src/app.js

```diff
@@ -25,6 +25,10 @@ var OffsetToLineColumnConverter = require('./lib/offsetToLineColumnConverter')
 
 var examples = require('./app/example-contracts')
 
+// components
+var fileExplorer = require('_file-explorer')
```

Comment:
One additional advantage i forgot to mention is, that - by default - browserify transforms only work on local files (thus files included in the current project), but not on `require(...)`ed modules files.

But, if a required file is written in - for example - coffee script, or uses some other babel features or transforms that inline assets into the built version, then those transforms need to be executed even though the current project doesnt know about it.

To enable this, a dependency (e.g. a local internal module) might specify browserify transforms inside it's related `package.json` file which will then be executed by browserify for only this dependent module :-) ...which allows module authors to specify all the transform needed.

I find this quite a neat feature - and this is why i usually stub a `package.json` for my internal modules, so that i can add custom transforms - e.g. for example for visual components that depend on certain fonts/images/etc... assets that i then either inline or otherwise "grab" without the main project's need for the knowledge of dependency specific "asset paths".

I can recommend the npm modules `urify` and/or `urify-emitter` in that context :-)